libs/ballot-interpreter/src/hmpb-rust/timing_marks.rs

```diff
@@ -527,14 +525,34 @@ fn score_timing_mark_geometry_match(
 pub fn find_actual_bottom_marks(
     complete_timing_marks: &Complete,
 ) -> Vec<Option<CandidateTimingMark>> {
-    const MIN_REQUIRED_MARK_SCORE: UnitIntervalScore = UnitIntervalScore(0.7);
-    const MIN_REQUIRED_PADDING_SCORE: UnitIntervalScore = UnitIntervalScore(0.5);
+    // allow marks at the exterior (i.e. corners) to be substantially cropped
+    const MIN_REQUIRED_EXTERIOR_MARK_SCORE: UnitIntervalScore = UnitIntervalScore(0.33);
+    const MIN_REQUIRED_EXTERIOR_PADDING_SCORE: UnitIntervalScore = UnitIntervalScore(0.5);
```

Comment:
I think I need to see the example ballot and the debug image to evaluate this change. And probably talk it through sync.

Reply:
See a partial debug image [on this comment](https://github.com/votingworks/vxsuite/pull/5633#discussion_r1852703642) for one of the motivating examples.